docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -225,5 +225,5 @@ AgroalDataSource dataSource2;
 
 == Agroal Configuration Reference
 
-include::generated/quarkus-agroal.adoc[]
+include::generated/quarkus-agroal.adoc[opts=optional]
```

Comment:
Another remark, I don't think the TOC level is OK, is it? Can you check that?

We should add one toc level with `leveloffset=+1`.